docs/introduction.rst

```diff
@@ -33,7 +33,7 @@ Image Saliency Maps: An Intuitive Introduction
 ----------------------------------------------
 
 Figure 4 shows a deep learning pipeline for recognizing objects in
-images; pixels in the image (in green) are processed by the pipeline
+images; pixels in the image (in green) are processed by the pipeline (in yellow)
```

Comment:
do we care about what the other colors (turquoise & red) represent in the diagram?

Reply:
No; I replaced the underlying image with an uncolored version for clarity.